src/test/setup-contracts-testcafe.mjs

```diff
@@ -0,0 +1,104 @@
+
+const defaultToken = {
+  accountKey: ACCOUNT_KEY,
+  tokenRegistryAddress: TOKEN_REGISTRY_ADDRESS,
+  owner: ADDRESS_EXAMPLE_1,
+  holder: ADDRESS_EXAMPLE_1,
+};
+
+const MERKLE_ROOT_OF_ENDORSEMENT_CHAIN_DOCUMENT = "0xbc08b23e18ae6028be5cd51f1fd7add305bb05448594326240817b7011e243da";
+
+// Prep for minting in token registry
+const merkleRootToMint = {
+  tokenRegistry: [
+    {
+      // ebl.json
+      merkleRoot: "0xd781801b86ed7bf603c5c6584bd50abab4e59e9c404ee18f3d12f191fea28133",
+      ...defaultToken,
+    },
+    {
+      //ebl-endorsement-chain.json
+      merkleRoot: MERKLE_ROOT_OF_ENDORSEMENT_CHAIN_DOCUMENT,
+      ...defaultToken,
+    },
+  ],
+};
+
+shell.exec(`${oaCLI_PATH} deploy title-escrow-factory -n local -k ${ACCOUNT_KEY}`);
+
+shell.exec(
+  `${oaCLI_PATH} deploy token-registry "DEMO TOKEN REGISTRY" DTR -n local -k ${ACCOUNT_KEY} --factory-address ${TITLE_ESCROW_FACTORY_ADDRESS} --standalone`
+);
+
+shell.exec(`${oaCLI_PATH} deploy document-store "My Document Store" -n local -k ${ACCOUNT_KEY}`);
+
+merkleRootToMint.tokenRegistry.forEach((element) => {
+  shell.exec(
+    `${oaCLI_PATH} token-registry issue --beneficiary ${element.owner} --holder ${element.holder} --address ${element.tokenRegistryAddress} --tokenId ${element.merkleRoot} -n local -k ${element.accountKey}`
+  );
+});
+
+shell.exec(
+  `${oaCLI_PATH} title-escrow nominate-change-owner --newBeneficiary ${ADDRESS_EXAMPLE_2} --token-registry ${TOKEN_REGISTRY_ADDRESS} --tokenId ${MERKLE_ROOT_OF_ENDORSEMENT_CHAIN_DOCUMENT} -n local -k ${ACCOUNT_KEY}`
+);
+shell.exec(
+  `${oaCLI_PATH} title-escrow endorse-transfer-owner --newBeneficiary ${ADDRESS_EXAMPLE_2} --token-registry ${TOKEN_REGISTRY_ADDRESS} --tokenId ${MERKLE_ROOT_OF_ENDORSEMENT_CHAIN_DOCUMENT} -n local -k ${ACCOUNT_KEY}`
+);
+shell.exec(
+  `${oaCLI_PATH} title-escrow change-holder --token-registry ${TOKEN_REGISTRY_ADDRESS} --tokenId ${MERKLE_ROOT_OF_ENDORSEMENT_CHAIN_DOCUMENT} --newHolder ${ADDRESS_EXAMPLE_2} -n local -k ${ACCOUNT_KEY}`
+);
+shell.exec(
+  `${oaCLI_PATH} title-escrow endorse-change-owner --newBeneficiary ${ADDRESS_EXAMPLE_1} --newHolder ${ADDRESS_EXAMPLE_1} --token-registry ${TOKEN_REGISTRY_ADDRESS} --tokenId ${MERKLE_ROOT_OF_ENDORSEMENT_CHAIN_DOCUMENT} -n local -k ${ACCOUNT_KEY_2}`
+);
+shell.exec(
+  `${oaCLI_PATH} title-escrow surrender --token-registry ${TOKEN_REGISTRY_ADDRESS} --tokenId ${MERKLE_ROOT_OF_ENDORSEMENT_CHAIN_DOCUMENT} -n local -k ${ACCOUNT_KEY}`
+);
+shell.exec(
+  `${oaCLI_PATH} title-escrow accept-surrendered --token-registry ${TOKEN_REGISTRY_ADDRESS} --tokenId ${MERKLE_ROOT_OF_ENDORSEMENT_CHAIN_DOCUMENT} -n local -k ${ACCOUNT_KEY}`
+);
+
+// prep for issuing document store
+const merkleRootToIssue = [
+  // demo document - local.tt
+  "0xd7a0c43098da5bcabd2093f6f1d8c35d6d898a2a0a17d92d3d1bf0c1845aa6a6",
+  // invoice-attachments.json
+  "0x927c044be8438b3f042ca4edb601269db115a67ae0bf5abeb9ae38695a002d4d",
+  // invoice-missing-renderer-url.json
```

Comment:
is there a way to import the hashes from the fixtures instead?

Reply:
would it be more troublesome?